### PR TITLE
chore(Makefile): updating echos and makefiles commands

### PR DESCRIPTION
**Pull request by reynan-dev**
## Ticket

**Title:**
**Notion Link:**

## Description

- Updating Makefile's command to fit both docker compose versions
- Updating echos to explain what the command are doing.

## Guidelines

🔗 Link your PRs to their related Notion tickets
✍️ Use the commit convention to your pull request title
📃 If your code affects the build, update `README.md`
🚫 Do not merge a PR until all comments are resolved
🗑 Remove your branch after merging

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,36 +1,45 @@
...
 start-server: ## Starting server in debug mode
+	@echo "Starting server in debug mode"
 	cd server && make start-debug
 
 start-database: ## Starting database in debug mode
+	@echo "Starting database in debug mode"
 	cd database && make start-debug
 
 init-database: ## Inizializate database with migrations
+	@echo "Inizializate database with migrations"
```
Comment (jkergal) [inline Makefile:32]:
```suggestion
	@echo "Initialize database with migrations"
```

File: Makefile [hunk 2]
```diff
@@ -1,36 +1,45 @@
...
 	cd server && make migration-run
 
 start-all: ## Starting everyservices in quiet mode.
+	@echo "Starting everyservices in quiet mode."
```
Comment (jkergal) [inline Makefile:36]:
```suggestion
	@echo "Starting every services in quiet mode."
```

File: Makefile [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
 	cd database && make start-quiet
 	cd server && make start-quiet
 	cd client && make start-quiet
 
 stop-all: ## Stoping everyservices
+	@echo "Stoping everyservices"
```
Comment (jkergal) [inline Makefile:42]:
```suggestion
	@echo "Stoping every services"
```

File: database/Makefile [hunk 4]
```diff
@@ -1,7 +1,7 @@
 start-quiet: ## Starting database (detached)
-	@echo "Initializating database - Docker compose v1"
+	@echo "Initializating database in quiet - Docker compose v1"
```
Comment (jkergal) [inline database/Makefile:2]:
```suggestion
	@echo "Initializating database in quiet mode - Docker compose v1"
```

**Review by jkergal — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (reynan-dev) [reply Makefile:32]: Done

File: Makefile (hunk 3 above)
Comment (reynan-dev) [reply Makefile:42]: Done

File: Makefile (hunk 2 above)
Comment (reynan-dev) [reply Makefile:36]: Done

File: database/Makefile (hunk 4 above)
Comment (reynan-dev) [reply database/Makefile:2]: Done

**Review by jkergal — APPROVED** [review]

**Pull request merged**